requirements.txt

```diff
@@ -41,11 +41,16 @@ PyISY>=1.0.2
 # sensor.systemmonitor 
 psutil>=2.2.1
 
-#pushover notifications
+# pushover notifications
 python-pushover>=0.2
 
 # Transmission Torrent Client
 transmissionrpc>=0.11
 
 # OpenWeatherMap Web API
-pyowm>=2.2.0
\ No newline at end of file
+pyowm>=2.2.0
+
+# XMPP Bindings (notify.xmpp)
+sleekxmpp>=1.3.1
+pyasn1>=0.1.7
```

Comment:
Are the dependencies of sleekxmpp not installed automatically when installing it?